src/controllers/users.controller.js

```diff
@@ -188,6 +188,24 @@ class UserController {
       res
     );
   }
+
+  /**
+   *function resend email
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns {object} responses
+   */
+  async fetchAllUsers(req, res) {
+    try {
+      const allUsers = await db.user.findAll({
+        attributes: { exclude: ['password'] }
+      });
+      return Responses.handleSuccess(200, 'successfully retrieved all users', res, allUsers);
+    } catch (error) {
```

Comment:
I think, the  `try - catch` is not need here, as we have this method is wrapped in the `catchErrors` in the route.

Reply:
Thanks for that. I have removed it.